docs/checkit.js

```diff
@@ -907,6 +981,16 @@ CheckIt.prototype.addCalendarToDropdown = function (uniqueId, title, $dropdown)
     //add the calendar with unique Id, uniqueId, and title, title, to
     //the saved calendars dropdown on the navbar.
     "use strict";
+    
+    // first check if the title isn't already in the dropdown
+    if ($('#' + uniqueId).length) {
```

Comment:
Why length?

Reply:
I'm not sure.

Reply:
I guess I can just take away .length and have 
`if ($('#' + uniqueId))`

right?

Reply:
I'm not sure. What does `$('#foo')` return if there is no element with id `foo` ? Does JS treat whatever that is as `true` when used in an `if` statement?

You can test it in the console pretty easily. You can also explain exactly why it's like this in more detail in the comment.

Reply:
Looks like `length` is indeed needed. Otherwise JS looks at the jquery object and objects are `true`.

```javascript
> $('#foo')
jQuery.fn.init {context: document, selector: "#foo"}
>$('#foo').length
0
> Boolean($('#foo'))
true
> Boolean($('#foo').length)
false
```